morpheus/_lib/include/morpheus/messages/meta.hpp

```diff
@@ -42,14 +43,14 @@ namespace morpheus {
  * @file
  */
 
-class MutableTableCtxMgr;
+MORPHEUS_EXPORT class MutableTableCtxMgr;
```

Comment:
Can we move `MORPHEUS_EXPORT` to be between `class` and `MutableTableCtxMgr` to be consistent?